docs/pywbemlistener/commands.rst

```diff
@@ -220,6 +220,11 @@ Example:
     $ pywbemlistener stop lis1
     Shut down listener lis1 running at http://localhost:25989
 
+On Windows, the stop of the listener process is immediate, without giving the
```

Comment:
grammar change.  On windows the listener process is stopped immediately without giving

Reply:
DONE, also in next sentence.